ethstorage/p2p/protocol/syncserver.go

```diff
@@ -79,9 +80,15 @@ func NewSyncServer(cfg *rollup.EsConfig, storageManager StorageManagerReader, m
 	if m == nil {
 		m = metrics.NoopMetrics
 	}
+
+	providedBlobs := make(map[uint64]uint64)
+	for _, shardId := range storageManager.Shards() {
+		providedBlobs[shardId] = 0
```

Comment:
Do we want to persist this data into the database? Why do we choose a different handling way with the mining status?